selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/CodenvyEditor.java

```diff
@@ -514,6 +514,17 @@ public void waitTextInHoverPopUpEqualsTo(String expectedText) {
     }
   }
 
+  /** wait full matching of text in hover popup */
+  public void waitTextInHoverPopUpEqualsTo(String expectedText, String knownIssue) {
+    // waits popup body visibility
+    try {
+      seleniumWebDriverHelper.waitVisibility(hoverPopup);
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
```

Comment:
this comment `// remove try-catch block after issue has been resolved` is irrelevant here.
Also, IMHO, it's better to catch exception inside the test - the place where failing test step is situated.